prow/config/config.go

```diff
@@ -864,21 +883,32 @@ func mergeJobConfigs(a, b JobConfig) (JobConfig, error) {
 	return c, nil
 }
 
-func setPresubmitDecorationDefaults(c *Config, ps *Presubmit) {
+func setPresubmitDecorationDefaults(c *Config, ps *Presubmit, repo string) {
 	if ps.Decorate {
-		ps.DecorationConfig = ps.DecorationConfig.ApplyDefault(c.Plank.DefaultDecorationConfig)
+		dcByRepo := c.Plank.GetDefaultDecorationConfigs(repo)
+		def := dcByRepo.ApplyDefault(c.Plank.DefaultDecorationConfig)
```

Comment:
You don't need these lines, since you add the default to the map

Reply:
addressed